test/irb/test_history.rb

```diff
@@ -17,8 +17,16 @@ def teardown
       IRB.conf[:RC_NAME_GENERATOR] = nil
     end
 
-    class TestInputMethodWithHistory < TestInputMethod
-      HISTORY = Array.new
+    class TestInputMethodWithRelineHistory < TestInputMethod
+      # When IRB.conf[:USE_MULTILINE] is true, IRB::ReidlineInputMethod uses Reline::History
```

Comment:
`IRB::ReidlineInputMethod` → `IRB::RelineInputMethod`
`IRB::ReidlineInputMethod` is not used anymore.

Other part of the change in `test_history.rb` looks great :+1:

Reply:
Thanks.  Updated.